lib/app/components/atoms/chat_bubble.dart

```diff
@@ -14,12 +16,15 @@ class ChatBubble extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BubbleSpecialThree(
-      text: text,
-      isSender: isSender,
-      color: colorChatBubbleBg, //TODO: 将来的にdark mode対応予定
-      tail: true,
-      textStyle: const TextStyle(color: colorChatBubbleText, fontSize: 16),
+    return cb.ChatBubble(
+      clipper: isSender
+          ? ChatBubbleClipper4(type: BubbleType.sendBubble)
+          : ChatBubbleClipper4(type: BubbleType.receiverBubble),
+      backGroundColor: colorChatBubbleBg,
+      child: Text(
+        text,
+        style: const TextStyle(color: colorChatBubbleText, fontSize: 16),
+      ),
```

Comment:
```suggestion
      child: Container(
        constraints:
            BoxConstraints(maxWidth: MediaQuery.of(context).size.width * 0.7),
        child: Text(
          text,
          style: const TextStyle(color: colorChatBubbleText, fontSize: 16),
        ),
      ),
```
これだとspeech_baloon幅が決まってなくて、改行されずにレイアウトエラーになってしまうので、
上記のように`BoxConstraints`を設けましょう。

Reply:
ご指摘の通り修正しました！